README.md

```diff
@@ -82,9 +82,13 @@ end
 ```
 
 ## Installation
+you can use it either downloading with curl,
```

Comment:
If you want to update the changelog and version, I will tag the merge commit appropriately.